src/templates/dataGridTableRow.jsx

```diff
@@ -27,30 +39,42 @@ const propTypes = {
 };
 
 const defaultProps = {
-    handle: undefined,
+    classes: null,
+    handle: false,
     isClickable: undefined,
     rowProps: undefined,
 };
 
-class DataGridTableRow extends React.PureComponent {
+const styles = {
+    handleGrid: {
+        flexWrap: 'nowrap !important',
+    },
+};
+
+class DataGridTableRow extends React.Component {
```

Comment:
Deliberately switched from `PureComponent` to `Component`...?

Reply:
Yes, no need to be PureComponent because of I believe `columns` and `data`.

Reply:
Resolving this conversation.